### DISCUSSION TITLE
Reading about workflows issues.... reporting

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16186] [How to]
It seems workflows causing some confusion (sometimes), so I would like to ask how would be the best way to debug / see outcome of all (or selected) workflows run in the system?

Something like:
Workflow name/ID
Trigger time
Trigger result (went OK or with ERRORS)
Trigger trace (why error, why workflow omitted etc)

Thanks!

**Comment by vovpff** [comment]
You can turn on system logs. Everything will be logged into cache/logs/ Here you can check situation around workflows

**Comment by ArniSamano** [comment]
The thing is I understand guys in other threads. I did some testing and situation seems confusing as the same workflow in manual trigger works fine but with execution schedule not.  Workflow in total consists of 5 conditions (AND)
So to understand what is going on, the full report of workflow triggered or omitted will be helpful. Logs don't show that
